lib/include.js

```diff
@@ -327,10 +404,12 @@ Inclusion.include = function (objects, include, options, cb) {
         throughFilter.where[polymorphic.discriminator] =
           relation.modelFrom.definition.name;
       }
+      findWithForeignKeysByPage(relation.modelThrough, throughFilter,
+        relation.keyTo, 0, options, throughFetchHandler);
       /**
        * 1st DB Call of 2 step process. Get through model objects first
        */
-      relation.modelThrough.find(throughFilter, options, throughFetchHandler);
+      // relation.modelThrough.find(throughFilter, options, throughFetchHandler);
```

Comment:
Commented-out code: if it's an intentional change, then please remove it completely.